--- a/opam
+++ b/opam
@@ -12,7 +12,6 @@ depends: [
   "ocamlfind" {build}
   "ocamlbuild" {build}
   "topkg" {build}
-  "uchar"
 ]
 depopts: ["cmdliner"]
 conflicts: ["cmdliner" { < "0.9.6"} ]
--- uutf-1.0.2/pkg/META.orig	2021-11-26 06:36:05.852321090 +0100
+++ uutf-1.0.2/pkg/META	2021-11-26 06:36:39.647405558 +0100
@@ -1,6 +1,6 @@
 version = "1.0.2"
 description = "Non-blocking streaming Unicode codec for OCaml"
-requires = "bytes uchar"
+requires = "bytes"
 archive(byte) = "uutf.cma"
 archive(native) = "uutf.cmxa"
 plugin(byte) = "uutf.cma"
--- a/_tags
+++ b/_tags
@@ -1,4 +1,4 @@
-true : bin_annot, safe_string, package(bytes), package(uchar)
+true : bin_annot, safe_string, package(bytes)
 
 <src> : include
 
